ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -1,214 +1,219 @@
 {# eos - Ethernet Interfaces #}
-{% if ethernet_interfaces is defined and ethernet_interfaces is not none %}
+{% if ethernet_interfaces is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed